spec/features/invites_feature_spec.rb

```diff
@@ -1,22 +1,24 @@
 require 'spec_helper'
 
-feature "invitations" do
-  given(:alice) { Fabricate(:user) }
+feature "invitations", {js: true, vcr: true } do
```

Comment:
Instead of `feature "invitations", {js: true, vcr: true } do`

you can use shorthand RSpec tags - `feature "invitations", :js, :vcr do`